packages/cli-kit/src/git.ts

```diff
@@ -60,6 +110,42 @@ export async function downloadRepository({
   }
 }
 
+export async function getLatestCommit(directory?: string) {
+  try {
+    const logs = await git({baseDir: directory}).log({
+      maxCount: 1,
+    })
+    if (!logs.latest) throw NoCommitError()
+    return logs.latest
+  } catch {
+    throw NoCommitError()
```

Comment:
Is this true? I'd let those errors bubble up and iterate on the catching logic here once we have more data.